docs/set_up_admin_tails.rst

```diff
@@ -164,13 +164,24 @@ in the SecureDrop repository that you just cloned.
 
 To use the template:
 
+-  Copy the template to the Persistent folder - from a terminal, run the
+   commands:
+
+   .. code:: sh
+
+     cd ~/Persistent
+     cp securedrop/tails_files/keepassx.kdbx keepassx.kdbx
```

Comment:
Should we use absolute paths here instead of cd'ing into the destination?

Reply:
Done. It makes the command a bit long, which is why I was initially against it, so it's now split across two lines.